docs/_interface/conf.md

```diff
@@ -0,0 +1,288 @@
+| UseLightweightKDF       | boolean       | If true, lowers the memory and CPU requirements of the key store scrypt KDF at the expense of security.
+| InsecureUnlockAllowed   | boolean       | Allow user to unlock accounts in unsafe http environment.
+| NoUSB                   | boolean       | Disable hardware wallet monitoring and connectivity
+| USB                     | boolean       | Enable hardware wallet monitoring and connectivity.
+| SmartCardDaemonPath     | string        | Path to the smartcard daemon's socket
+| IPCPath                 | string        | Requested location to place the IPC endpoint.
+| HTTPHost                | string        | Host interface on which to start the HTTP RPC server
+| HTTPPort                | int           | TCP port for HTTP RPC server. Zero means to pick a port number randomly
+| HTTPCors                | string array  | Cross-Origin Resource Sharing header to send to requesting clients (which may or may not obey it)
+| HTTPVirtualHosts        | string array  | List of virtual hostnames which are allowed on incoming requests	
+| HTTPModules             | string array  | List of API modules to expose via the HTTP RPC interface
+| HTTPPathPrefix          | string        | Path prefix on which http-rpc is to be served.
+| WSHost                  | string        | Host interface on which to start the websocket RPC server
+| WSPort                  | int           | TCP port for websocker RPC server. Zero means to pick a port number randomly
+| WSPathPrefix            | string        | Path prefix on which ws-rpc is to be served.
+| WSOrigins               | string array  | List of domain to accept websocket requests from
+| WSModules               | string array  | List of API modules to expose via the websocket RPC interface.
+| WSExpose                | boolean       | Expose all API modules via the WebSocket RPC interface rather than just the public ones.  **Only for trusted networks**
+| GraphQLCors             | string array  | Cross-Origin Resource Sharing header to send to requesting clients (which may or may not obey it)
+| GraphQLVirtualHosts     | string array  | List of virtual hostnames which are allowed on incoming requests.
+| AllowUnprotectedTxs     | boolean       | Allow non [EIP-155 protected transactions](https://github.com/ethereum/EIPs/blob/master/EIPS/eip-155.md) to be sent over RPC
+	
+  
+
+### Node.P2P Settings
+
+These are peer to peer network settings. [The configuration object is documented here](https://pkg.go.dev/github.com/ethereum/go-ethereum@v1.10.4/p2p#Config)
+
+
+| Settings                | Type          | Meaning                                                                                              |
+| ----------------------- | ------------- | ---------------------------------------------------------------------------------------------------- |
+| PrivateKey              | [PrivateKey](https://pkg.go.dev/crypto/ecdsa#PrivateKey) | The private key for the node |
```

Comment:
I think you can either specify `nodekey` to load a pk from file or `nodekeyhex` to specify the nodekey as a hex string

Reply:
How? Is the value either

```
nodekey:/path/to/file.pk
```

or

```
nodekeyhex: 0xDEADBEEF
```

?